src/clinical/clinical-service.ts

```diff
@@ -53,13 +54,14 @@ export type ClinicalDonorEntityQuery = {
   completionState?: {};
 };
 
-// Returns paginated Donor + Entity Submission Data
-export type PaginatedClinicalQuery = ClinicalDonorEntityQuery & {
+export type PaginationQuery = {
   page: number;
   pageSize?: number;
   sort: string;
 };
 
+export type PaginatedClinicalQuery = ClinicalDonorEntityQuery & PaginationQuery;
```

Comment:
We had a need to separate the a worker thread task to accept Pagination parameters separate from the Clinical Donor Entity query data, so we can separate out that data for reference to its own type `PaginationQuery`.